### PR TITLE
Add "Debug logging" advanced setting

### PR DESCRIPTION
**Pull request by Hocuri**
To be merged after https://github.com/deltachat/deltachat-core-rust/pull/3296

- [x] localization

### CODE REVIEW

File: res/xml/preferences_advanced.xml [hunk 1]
```diff
@@ -89,5 +89,11 @@
 
         <Preference android:key="pref_view_log"
                     android:title="@string/pref_view_log"/>
+
+        <org.thoughtcrime.securesms.components.SwitchPreferenceCompat
```
Comment (adbenitez) [inline res/xml/preferences_advanced.xml:93]: can't we reuse the developer option for this? some extra debugging that makes the app slower and perhaps unreliable sounds like a perfect fit :D

Comment (Hocuri) [reply res/xml/preferences_advanced.xml:93]: Hm, then we don't warn the user as explicitly about the slowndown, and we can't disable the option when the webxdc is deleted, so this "the webxdc is there exactly when the option is on" relationship will be gone.

And the developer option is for people who develop webxdcs (at least currently) while the debug logging option is for people who want to track down issues with deltachat core, which doesn't necessarily belong together.

**Review by adbenitez — APPROVED** [review]
a pity this makes things slower if enabled, hope that gets better in the future :(

**Comment by Hocuri** [comment]
Seems not to be necessary after all.

**Pull request closed without merging**